File: Java/Booking/SubmitOrderOnAccountResult.java
package LTD_API;

public class SubmitOrderResult
{
    private bool success;
    private String bookingReference;
    
    public bool getSuccess()
    {
        return success;
    }
    public void setSuccess(bool success)
    {
        this.success = success;
    }

    public String getBookingReference()
    {
        return bookingReference;
    }
    public void setBookingReference(String bookingReference)
    {
        this.bookingReference = bookingReference;
    }
}

File: PHP/Booking/SubmitOrderOnAccountResult.php
<?php
namespace LTD_API;

class SubmitOrderResult
{
    private $success;
    private $bookingReference;
    
    public function getSuccess()
    {
        return $this->success;
    }
    public function setSuccess($success)
    {
        $this->success = $success;
    }

    public function getBookingReference()
    {
        return $this->bookingReference;
    }
    public function setBookingReference($bookingReference)
    {
        $this->bookingReference = $bookingReference;
    }
}

